DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -176,7 +185,10 @@ public IActionResult GroupCourses(int groupId, int page = 1)
                 return NotFound();
             }
 
-            var groupCourses = groupsDataService.GetGroupCourses(groupId, centreId);
+            var adminId = User.GetAdminId()!.Value;
+            var adminUser = userDataService.GetAdminUserById(adminId)!;
+
+            var groupCourses = groupsService.GetGroupCoursesForCategory(groupId, centreId, adminUser.CategoryIdFilter);

```

Comment:
Is there a reason to use this over User.GetAdminCategoryId()?

Reply:
This property handles the conversion of category 0 to null, which is a convention we're using for handling admin category restrictions in SQL. I'll DM you a link to the recent slack conversation.

Reply:
Dan is making a change to the claim in HEEDLS-564 https://github.com/TechnologyEnhancedLearning/DLSV2/pull/698 (tech lead review). Will move this over to tech lead review for now, but if that does get merged, I think we should update to using the claims version.